sdks/node-sdk/src/Group.ts

```diff
@@ -80,16 +81,16 @@ export class Group extends Conversation {
     return this.#conversation.isSuperAdmin(inboxId);
   }
 
-  async addMembers(accountAddresses: string[]) {
-    return this.#conversation.addMembers(accountAddresses);
+  async addMembers(identifiers: Identifier[]) {
+    return this.#conversation.addMembers(identifiers);
   }
 
   async addMembersByInboxId(inboxIds: string[]) {
```

Comment:
You forgot to swap these. addMembers and removeMembers should be inboxId and the other type should take an identifier. Can you address in a follow up. 🙏 @rygine